app/Http/Controllers/UsersController.php

```diff
@@ -420,7 +369,7 @@ public function destroy($id = null)
 
             // Redirect to the user management page
             return redirect()->route('users.index')->with('success', $success);
-        } catch (UserNotFoundException $e) {
+        } catch (ModelNotFoundException $e) {
```

Comment:
Why model not found here?

Reply:
As far as I can tell, UserNotFoundException doesn't exist anywhere... Maybe it was a laravel 4 relic?

Reply:
I can't remember exactly what led to this change--It was a few months ago, I remember that  I didn't hit the expected error until changing it to this approach. 

There are a number of other places in UsersController that uses UserNotFoundException, I haven't looked into porting those.